geometry:
  r: 0.03425
  d: 0.173
  cr: 0.0125
battery:
  design_capacity: 1.3
  serial_number: ''
motors:
  # True Gear Ratio (22×20×22×22×23) / (12×12×10×10×10) ~= 34.014
  gear_ratio: 34.014
  encoder_ratio: 48
  max_rpm: 290
  min_volts: 0.0
  max_volts: 12.0
  max_cmd: 400
  kp: 0.1
  ki: 0.0
  kd: 0.0
controller:
  leds: 3
  buttons: 3
  notes: 16
  update_rate: 50.0
  position_sample_rate: 50.0
  button_sample_rate: 4.0
  battery_sample_rate: 0.2
  cmd_timeout: 0.10


